### PR TITLE
Dropped calls to insserv

### PR DESCRIPTION
**Pull request by mchf**
[bsc#1026027](https://bugzilla.suse.com/show_bug.cgi?id=1026027)

### CODE REVIEW

**Comment by mchf** [comment]
@charlesa 
Could you pls review? Thanks

File: src/clients/vm_finish.rb [hunk 1]
```diff
@@ -81,26 +82,9 @@ def main
...
-              "insserv: service %1 disabled: %2",
-              s,
-              disabled == 0 ? true : false
-            )
-          end 
-
-
-          # We no longer need to do remove these entries now that we use xvc
-          #SCR::Execute(.target.bash, "/bin/sed -i 's/.*mingetty tty[2-9]$//' /etc/inittab");
+          @disable_services.all { |s| Service.Disable(s) }
```
Comment (mvidner) [inline src/clients/vm_finish.rb]:
```console
$ ruby -e '[].all'
-e:1:in `<main>': undefined method `all' for []:Array (NoMethodError)
```

(This bug would be discovered by static analysis)

File: package/yast2-vm.changes [hunk 2]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Mar 29 10:24:09 UTC 2017 - mfilka@suse.com
+
+- bnc#1026027
+  - droppet calls to insserv
```
Comment (mvidner) [inline package/yast2-vm.changes]: Droplet. I mean, dropped.

File: src/clients/vm_finish.rb (hunk 1 above)
Comment (mchf) [inline src/clients/vm_finish.rb]: in fact there should be ```each``` I've obviously didn't push the most recent version :-/

**Pull request merged**